web/impact/impact/v1/views/utils.py

```diff
@@ -0,0 +1,15 @@
+def merge_data_by_id(data):
+    result = {}
+    for datum in data:
+        id = datum["id"]
```

Comment:
using builtin keyword "id" - rename to _id

Reply:
Good point.  Ended up renaming the function and generalizing `id` to an argument called `merge_field` which defaults to "id".